package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "deadfile",
+  "name": "@foundation-lib/deadfile",
```

Comment:
Could you please change this back to deadfile?

Reply:
Yep; sorry; should have done that right away after publishing to my private scope